com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/MeshDeformation/ImposterSampleNode.cs

```diff
@@ -0,0 +1,126 @@
+                new string[]
+                    {
+                    "\"Packages/com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/MeshDeformation/Imposter.hlsl\""
+                    }
+            )
+        }
+    );
+
+        public static NodeUIDescriptor NodeUIDescriptor => new(
+            Version,
+            Name,
+            displayName: "Imposter Sample",
+            tooltip: "Samples from the three virtual UVs and blends them base on the camera intersection point to get the correct result.",
+            category: "Input/Mesh Deformation",
+            hasPreview: false,
+            description: "pkg://Documentation~/previews/ImposterSample.md",
+            synonyms: new string[] { "billboard" },
+            selectableFunctions: new()
+        {
+            { "ThreeFrames", "Three Frames" },
+            { "OneFrame", "One Frame" }
+        },
+            functionSelectorLabel: "Sample Type",
+            parameters: new ParameterUIDescriptor[] {
+                new ParameterUIDescriptor(
+                    name: "Texture",
+                    tooltip: "The texture asset to sample"
+                ),
+                new ParameterUIDescriptor(
+                    name: "Sampler",
+                    tooltip: "The texture sampler to use for sampling the texture"
+                ),
+                new ParameterUIDescriptor(
+                    name: "UV0",
+                    tooltip: "The virtual UV for the base frame"
+                ),
+                new ParameterUIDescriptor(
+                    name: "UV1",
+                    tooltip: "The virtual UV for the second frame"
+                ),
+                new ParameterUIDescriptor(
+                    name: "UV2",
+                    tooltip: "The virtual UV for the third frame"
+                ),
+                new ParameterUIDescriptor(
+                    name: "Grid",
+                    tooltip: "The current UV grid"
+                ),
+                new ParameterUIDescriptor(
+                    name: "Frames",
+                    tooltip: "The amount of the imposter frames"
+                ),
+                new ParameterUIDescriptor(
+                    name: "Clip",
+                    tooltip: "The amount of clipping for a single frame"
+                ),
+                new ParameterUIDescriptor(
+                    name: "Parallax",
+                    tooltip: "Adds parallax effect the port value is true"
+                ),
+                new ParameterUIDescriptor(
+                    name: "HeightMapChannel",
+                    displayName:"Heigh Map Channel",
+                    tooltip: "The texture channel to sample from for the parallax effect"
+                ),
+                new ParameterUIDescriptor(
+                    name: "RGBA",
+                    tooltip: "A vector4 from the sampled texture"
+                )
+            }
+        );
+    }
+}
```

Comment:
If the imposter texture is a normal map, what needs to happen to the normal map data after it's sampled in order for the output normal to match the normals of the actual object? Is that something that this node could handle, or do you think it should be done separately? Our Sample Texture 2D node handles normal maps. Do you think this one should follow that pattern?